core/modules/node/node.content_block.inc

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/**
+ *  ContentBlock extends Block
+ *  
+ *  This class allows us to display a particular node in a block.
+ */
+class ContentBlock extends Block {
```

Comment:
Although we use "Content" on the front-end, sticking to our convention of "Node" is good for namespacing this class. Could we rename to `NodeBlock`?